.travis.yml

```diff
@@ -101,12 +101,16 @@ script:
   - |
     if [[ -v HIVE_TESTS ]]; then
       presto-hive-hadoop2/bin/run_hive_tests.sh
-      if [[ -v S3_TESTS_AWS_ACCESS_KEY_ID ]]; then
-        env AWS_ACCESS_KEY_ID=$S3_TESTS_AWS_ACCESS_KEY_ID \
-        AWS_SECRET_ACCESS_KEY=$S3_TESTS_AWS_SECRET_ACCESS_KEY \
+      if [[ -v HIVE_TESTS_AWS_ACCESS_KEY_ID ]]; then
```

Comment:
should we different key for glue tests? We internally run S3 tests. Would the same key work for glue test even though we didn't setup anything specifically for glue?

Reply:
sounds good. We can stick with single key then? Are glue API calls enabled by default?

Reply:
> Are glue API calls enabled by default?
Not sure, you would have to check the IAM policies for that access key.